app/templates/source/sass/blocks/_forms.scss.ejs

```diff
@@ -336,7 +343,14 @@
 		.field {
 			> label,
 			> .group-label {
-				@extend %a11y;
+				overflow: hidden;
```

Comment:
Try not to use `overflow: hidden` at all please, it cuts off possible styling (e.g. box-shadow etc.).